src/repr/src/row.proto

```diff
@@ -102,7 +102,7 @@ message ProtoArray {
 }
 
 message ProtoArrayDimension {
-    uint64 lower_bound = 1;
+    int64 lower_bound = 1;
```

Comment:
Are we certain there are no values that would now overflow? Ideally we validate this on all data that we store.

Reply:
Yeah, there's just literally no code path right now that sets `lower_bound` to anything besides `1`, as far as I can find.